server/src/main/resources/application.properties

```diff
@@ -1 +1 @@
-
```

Comment:
I would rather version an application.properties file with empty key such as
`jwt.key=`
So that in development we can all set it and in production it does not get pushed